ParticleNet/python/trainSglConfigForGA.py

```diff
@@ -90,7 +90,7 @@ def train(model, optimizer, scheduler, use_plateau_scheduler=False):
 
     total_loss = 0.
     for data in trainLoader:
-        transform_data(data.to(args.device))
+        data = transform_data(data.to(args.device))
```

Comment:
We don't need to return data back if the function is called by reference. I would suggest to remove return and make it a void function for transform_data